light_curves/multiband_lc.ipynb

```diff
@@ -591,7 +591,7 @@
    "source": [
     "%%time\n",
     "\n",
-    "for objectid, coord in coords_list:\n",
+    "for objectid, coord in coords_list[4:5]:\n",
```

Comment:
Was this an accidental commit?

Reply:
yes that was accidental. I think it was already resolved in another push